### PR TITLE
Bmp280 example

### PR DESCRIPTION
**Pull request by Citrullin**
Updated the BMP280 example.

### CODE REVIEW

File: examples/BME280/BME280.ino [hunk 1]
```diff
@@ -1,40 +1,11 @@
...
- * WiFi using Microchip (Atmel) WINC1500
- * LED attached to pin 1 (onboard LED) for SAMW25
- * LED attached to pin 6 for MKR1000
-
- created 25 Nov 2012
- by Tom Igoe
-
- updates: dh, kg 2018
- */
-
```
Comment (relu91) [inline examples/BME280/BME280.ino:29]: Why removing this header description? Can't you just update it?

File: examples/BME280/BME280.ino [hunk 2]
```diff
@@ -79,49 +66,41 @@ void readBME280Data() {
...
-    digitalWrite(LED_BUILTIN, PIN_STATE_HIGH);
-    delay(500);
-    digitalWrite(LED_BUILTIN, PIN_STATE_LOW);
-    delay(500);
-  }
+  Serial.println("BME280I2C connected and initialized.");
 
   // connected, make the LED stay on
-  digitalWrite(LED_BUILTIN, PIN_STATE_HIGH);
+  //digitalWrite(LED_BUILTIN, PIN_STATE_HIGH);
```
Comment (relu91) [inline examples/BME280/BME280.ino]: Why don't we keep this visual feedback?

**Review by relu91 — COMMENTED** [review]
Looks good, I had just some questions below:

File: examples/BME280/BME280.ino (hunk 1 above)
Comment (Citrullin) [reply examples/BME280/BME280.ino:29]: I am going to write a full markdown file on this.

File: examples/BME280/BME280.ino (hunk 2 above)
Comment (Citrullin) [reply examples/BME280/BME280.ino]: Fair enough. Don't remember why I commented it out ^^

Comment (Citrullin) [reply examples/BME280/BME280.ino]: Fixed it.

File: examples/BME280/BME280.ino (hunk 1 above)
Comment (relu91) [reply examples/BME280/BME280.ino:29]: Do you want to do that in a subsequent PR? if so I'll add an issue after merging.

**Review by relu91 — APPROVED** [review]

File: examples/BME280/BME280.ino (hunk 1 above)
Comment (Citrullin) [reply examples/BME280/BME280.ino:29]: Yes, already created an issue for that. 
https://github.com/relu91/webthing-arduino/issues/9

**Pull request merged**